src/ocrd_models/ocrd_mets.py

```diff
@@ -267,8 +267,8 @@ def find_files(
             local_filename (string) : ``@xlink:href`` local/cached filename of ``mets:Flocat`` of ``mets:file``
             mimetype (string) : ``@MIMETYPE`` of ``mets:file``
             local (boolean) : Whether to restrict results to local files in the filesystem
-            include_fileGrp (list[str]) : Whitelist of allowd file groups 
-            exclude_fileGrp (list[str]) : Blacklist of disallowd file groups 
+            include_fileGrp (list[str]) : List of allowed file groups
```

Comment:
Is this because of white/black being problematic as terms? Because if it is just that `whitelist`/`blacklist` are not genuine dictionary-listed English terms, I'd prefer to keep them.

Reply:
Initially the lines were changed because of a typo, but while fixing it I decided to avoid the unnecessary color attributes as well because they are problematic for some people.

Reply:
Spell checkers like `codespell` and `typos` do not complain about colored lists.